server/bin/www

```diff
@@ -12,7 +12,7 @@ var http = require('http');
  * Get port from environment and store in Express.
  */
 
-var port = normalizePort(process.env.PORT || '3000');
+var port = normalizePort(process.env.PORT || '9000');
```

Comment:
We should leave this at the default of 3000 IMO

Reply:
Hmm are you sure they can both run on port 3000

Reply:
Well, they can't.  But the frontend and backend use separate development servers, and the proxy configuration on the frontend expects the server to be on port 3000.  Then you run the frontend with a PORT=3001 environment variable so there's no conflict.